config/build_deps.mk

```diff
@@ -17,10 +17,26 @@ ifeq ($(ENABLE_CARDINAL),yes)
   # Add DagMC flags (-DDAGMC is used in OpenMC)
   libmesh_CXXFLAGS    += -DENABLE_DAGMC -DDAGMC
 
+  # Disable Double-Down (optional dependency of DagMC) by default for now
+  # (see neams-th-coe/cardinal#1142). This also turns off Embree by default, below.
```

Comment:
```suggestion
  # (see https://github.com/neams-th-coe/cardinal/pull/1142). This also turns off Embree by default, below.
```

Reply:
I don't think this is necessary? We make comments like this often.

Reply:
Yeah, I don't think this is necessary.

Reply:
As discussed privately with @cticenhour, it's indeed not a necessary change for sure, but I think it increases convenience since it's an easier copy-paste for whoever is trying to learn more.